hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/ContainerSchemaDefinition.java

```diff
@@ -80,7 +80,7 @@ public void initializeSchema() throws SQLException {
   private void createUnhealthyContainersTable() {
     dslContext.createTableIfNotExists(UNHEALTHY_CONTAINERS_TABLE_NAME)
         .column(CONTAINER_ID, SQLDataType.BIGINT.nullable(false))
-        .column(CONTAINER_STATE, SQLDataType.VARCHAR(16).nullable(false))
+        .column(CONTAINER_STATE, SQLDataType.VARCHAR(40).nullable(false))
```

Comment:
Please check for compatibility for existing DB, we may need change column length on upgrade.